src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const PRICE = 40;
+  let sale = 0;
```

Comment:
Use `const` instead of `let` for the `sale` variable. Since `sale` is assigned a value and never reassigned, `const` is more appropriate and safe to use here.

Reply:
but in a second condition I reassign it if its LONG_TERM